build.gradle

```diff
@@ -46,10 +46,6 @@ test {
   }
 }
 
-task wrapper(type: Wrapper) {
-  gradleVersion = '4.0'
-}
-
```

Comment:
The build-in task cannot be overridden any more. https://docs.gradle.org/5.2.1/userguide/upgrading_version_4.html#deprecations_4.8 This PR needs to fix this.

The built-in wrapper task allows us to configure "wrapper" task's behavior. https://docs.gradle.org/6.9/userguide/gradle_wrapper.html#customizing_wrapper
Specifying the Gradle version here for "./gradlew wrapper" while `gradle/wrapper/gradle-wrapper.properties` gives a version does not make sense. So I'm removing these lines.